radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/module/ArchiveEndpointModule.java

```diff
@@ -125,6 +125,13 @@ public JsonRpcHandler accountGetUnstakePositions(ArchiveAccountHandler archiveAc
 		return archiveAccountHandler::handleAccountGetUnstakePositions;
 	}
 
+	@ArchiveEndpoint
+	@ProvidesIntoMap
+	@StringMapKey("account.get_transaction_history2")
```

Comment:
Can we write a more descriptive name describing the difference between this method and the existing `account.get_transaction_history`?

Reply:
Yes, this will be resolved in a future PR.